JavaScriptLogicGames/MineSweeper_puzzle/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Minesweeper Puzzle</title>
+        <link rel="stylesheet" type="text/css" href="/styles.css" />

```

Comment:
Bug. 
While referencing file "/styles.css" use "./styles.css"
Except your local machine no other user can access the file.